test/automigrate.test.js

```diff
@@ -0,0 +1,29 @@
+// Copyright IBM Corp. 2015,2016. All Rights Reserved.
+// Node module: loopback-connector-cloudant
+// This file is licensed under the Artistic License 2.0.
+// License text available at https://opensource.org/licenses/Artistic-2.0
+
+'use strict';
+var db, Foo, Bar;
+
+describe('cloudant imported features', function() {
+  before(function() {
+    require('./init.js');
+  });
+  it('automigrates models attached to db', function(done) {
+    db = getSchema();
+    Foo = db.define('Foo', {
+      name: {type: String},
+    });
+    Bar = db.define('Bar', {
+      id: {type: Number, index: true},
+    });
+    db.automigrate(function createSampleData() {
```

Comment:
I like the idea. It accurately describes what the function's purpose is.
(Of course, you should never take for granted that what the function _says_ it does is what it's actually doing, but it's still helpful.)

I would avoid naming the function definition on the right-hand side of the assignment operator if the left-hand was named (`Foo.prototype.createWidget = function createWidget() { ... }`), but this isn't one of those cases.